code/lib/csf-tools/src/CsfFile.ts

```diff
@@ -578,14 +604,16 @@ export class CsfFile {
   }
 }
 
-export const loadCsf = (code: string, options: CsfOptions) => {
+export const loadCsf = (code: string, originalCode: string, options: CsfOptions) => {
```

Comment:
if this is necessary, can we please make `originalCode` an optional field on the `options` object? this is a very disruptive PR otherwise, and `loadCsf` is also used in other packages outside of the monorepo